internal/eval/commands.go

```diff
@@ -926,6 +926,30 @@ var (
 		Arity:    -2,
 		KeySpecs: KeySpecs{BeginIndex: 1},
 	}
+	bitfieldCmdMeta = DiceCmdMeta{
+		Name: "BITFIELD",
+		Info: `The command treats a Redis string as an array of bits, 
```

Comment:
Hey @apoorvyadav1111 , do we need to mention string as "**Redis** string" here?

Reply:
Thanks @iamskp11 , I ll fix this.